prusti-viper/src/encoder/procedure_encoder.rs

```diff
@@ -2361,7 +2365,7 @@ impl<'p, 'v: 'p, 'tcx: 'v> ProcedureEncoder<'p, 'v, 'tcx> {
         Ok(result)
     }
 
-    fn encode_slice_len_call(
+    fn len(
```

Comment:
This name is not descriptive enough.

Reply:
I think this was an accidental change due to a replace-all